runFunctionalTestsDependencyCollector.cmd

```diff
@@ -6,10 +6,8 @@ CALL buildReleaseFull.cmd
 set BuildRoot=%~dp0..\bin\Release\
 set VSTestPath=%PROGRAMFILES(x86)%\Microsoft Visual Studio\2017\Enterprise\Common7\IDE\CommonExtensions\Microsoft\TestWindow\vstest.console.exe
 
-CALL "%VSTestPath%" "%BuildRoot%Test\E2ETests\E2ETests\DependencyCollectionTests.dll" /TestCaseFilter:"TestCategory=Core20" /logger:trx
```

Comment:
did you intend to make this change? or accidently checked in!

Reply:
oh, did not intend to remove those